librz/util/hex.c

```diff
@@ -394,21 +394,21 @@ RZ_API int rz_hex_bin2str(const ut8 *in, int len, char *out) {
 }
 
 /**
- * \brief Takes an unsigned integer and returns the signed integer in hex format as string.
- * E.g.: 0xffffffffffffffff -> "-0x1"
+ * \brief Takes an unsigned 32bit integer with MSB set to 1 and returns the signed integer in hex format as string.
+ * E.g.: 0xffffffff -> "-0x1"
  *
  * \param in The integer to convert to the signed string.
  * \param out The buffer to write the signed hex string to.
  * \param len Length of the out buffer.
  * \return char* The signed integer as hex string.
  */
 RZ_API void rz_hex_ut2st_str(const ut64 in, RZ_INOUT char *out, const int len) {
```

Comment:
Should the type of the argument also change then?

Reply:
Yes it should 24b28c1bd31226ad4c53302dcc56c3cb5527de11